keyboards/keychron/k1/rgb/se/config.h

```diff
@@ -40,7 +40,7 @@
 #define DEBOUNCE 5
 
 /* LED Status indicators */
-#define LED_CAPS_LOCK_PIN D4
+#define LED_CAPS_LOCK_PIN B12
```

Comment:
The caps lock indicator LED was working fine on my keyboard, seems like your keyboard uses a different pin. Probably need to move this into the mechanical/optical config files so it changes depending on the keyboard chosen. I can fix this after merging if you'd like?

Reply:
I might as well do that now, seems like the best option.